src/main/java/org/motechproject/prevac/repository/VisitDataService.java

```diff
@@ -261,4 +261,46 @@ long countFindByClinicIdVisitTypeAndPlannedVisitDate(
             @LookupField(name = "type") Set<VisitType> typeSet,
             @LookupField(name = "dateProjected") Range<LocalDate> plannedDate);
 
```

Comment:
I think you can remove all of the lookups above that ends with "AndVisitTypeSetAndPlannedDate"